src/index.test.ts

```diff
@@ -1,17 +1,91 @@
 import Geckoboard from './index';
-import { MockAgent, setGlobalDispatcher } from 'undici';
+import { MockAgent, setGlobalDispatcher, Interceptable } from 'undici';
 
+const prepareDatasetForCreation = (mockPool: Interceptable) => {
```

Comment:
🏄 I tend to prefer putting these at the bottom of the file as they're details to be read later rather be read first